learn/advanced/prefix.md

```diff
@@ -2,7 +2,7 @@
 
 In MeiliSearch, **you can perform a search with only a single letter as your query**. This is because we follow the philosophy of **prefix search**.
 
-Prefix search is when document sorting starts by comparing the search query against the beginning of each word in your dataset. All documents with words that match the query term are added to the [bucket sort](/reference/under_the_hood/bucket_sort.md), before the [ranking rules](/learn/core_concepts/relevancy.md#ranking-rules) are applied sequentially.
+Prefix search is when document sorting starts by comparing the search query against the beginning of each word in your dataset. All documents with words that match the query term are added to the [bucket sort](/learn/advanced/bucket_sort.md), before the [ranking rules](/learn/core_concepts/relevancy.md#ranking-rules) are applied sequentially.
```

Comment:
I don't see the bucket sort article under Advanced topics

Reply:
Yeah, I have just removed it, but forgot to sort out dead links. As suggested by @dichotommy, I will just point those to a wikipedia article.

Reply:
I'd suggest going for something other than Wikipedia. Idk if its just me but I don’t think its always reliable 👀

Reply:
If we're looking for an alternative to Wikipedia, I like this one: https://www.bbc.co.uk/bitesize/guides/z2m3b9q/revision/3

Reply:
The nice thing about wikipedia is that it's basically guaranteed to always be up, and links don't change frequently. Even if the info isn't necessarily reliable, it's a very reliable place to link to and is generally seen as "neutral" (even if that's not the case).

I think for now it's okay to do a quick google search and choose what we identify as the best resource. However, at some point in the future we may want to formalize our approach to external links.